Clinical-Genomics/scout
# Persistent filters

This PR adds a functionality: users can save filter settings that can be accessed by the institute

<img width="1600" alt="Screenshot 2019-10-27 at 21 33 20" src="https://user-images.githubusercontent.com/758570/67641278-84cfb880-f901-11e9-8b0f-248f1da30560.png">

I'm very tempted to use these saved filters to document different analysis sub-tasks. E.g. to log into the report that the user has checked SNVs for AR, de novo and reduced penetrance AD, and STRs but not yet SVs. But, that is likely another PR.

Also, a filter list with remove options should be present on the upcoming institute page.

**How to test**:
For each of the SNV, SV and Cancer variants filters,
1. Edit the filter settings and hit save. 
1. Ensure that a prompt asks for a filter name
1. Make sure you can load the filter, e.g. by hitting Reset filters, selecting your saved filter and hitting Load filter again.

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
